### PR TITLE
misc clean

### PR DESCRIPTION
**Pull request by t-h2o**
#### Misc clean

- Makefile: sort by alphabet
- `class_launcher` -> `launcher_class`
- tdd: header update

#### Waiting on

* [x] #23

#### tips

to replacing `value_launcher` to `launcher_value` and other launcher prototype, I used this command:

```bash
sed -i '' 's/\([a-z]*_*[a-z]*\)_launcher/launcher_\1/' files
```
the `()` is a group, there is one groupe, and for calling the group, I use `\1`

### CODE REVIEW

**Review by RobinBurri — APPROVED** [review]
RAS

**Comment by 0xCAF3D0OD** [comment]
what is the purpose of this PR ?!

**Comment by t-h2o** [comment]
> what is the purpose of this PR ?!

To make the code **prettier**

#### Examples:

Which one do you prefer ?

**before:**

```c
result = arguments_launcher();
result |= http_request_launcher();
result |= value_launcher();
result |= http_response_launcher();
result |= status_code_launcher();
```

**after:**

```cpp
result = launcher_arguments();
result |= launcher_http_request();
result |= launcher_value();
result |= launcher_http_response();
result |= launcher_status_code();
```

And for the `Makefile` ?

**before:**

```make
SRCS	=	$(SRCD)/main.cpp
SRCS	+=	$(SRCD)/webserver.cpp
SRCS	+=	$(SRCD)/arguments/check_argc_number.cpp
SRCS	+=	$(SRCD)/http/Request.cpp
SRCS	+=	$(SRCD)/json/Value.cpp
SRCS	+=	$(SRCD)/http/Response.cpp
SRCS	+=	$(SRCD)/http/StatusCodes.cpp
SRCS	+=	$(SRCD)/std98/std98.cpp
```

**after:**

```Makefile
SRCS    =       $(SRCD)/main.cpp
SRCS    +=      $(SRCD)/webserver.cpp
SRCS    +=      $(SRCD)/arguments/check_argc_number.cpp

# namespace http

SRCS    +=      $(SRCD)/http/Request.cpp
SRCS    +=      $(SRCD)/http/Response.cpp
SRCS    +=      $(SRCD)/http/StatusCodes.cpp

# namespace json

SRCS    +=      $(SRCD)/json/Value.cpp

# namespace std98

SRCS    +=      $(SRCD)/std98/std98.cpp

```

File: test/src/main.cpp [hunk 1]
```diff
@@ -5,11 +5,11 @@ main(void)
...
-	result = arguments_launcher();
-	result |= http_request_launcher();
-	result |= value_launcher();
-	result |= http_response_launcher();
-	result |= status_code_launcher();
+	result = launcher_arguments();
+	result |= launcher_http_request();
+	result |= launcher_value();
+	result |= launcher_http_response();
+	result |= launcher_status_code();
```
Comment (RobinBurri) [inline test/src/main.cpp:12]: launch vs launcher

**Review by RobinBurri — APPROVED** [review]
OK

File: test/src/main.cpp (hunk 1 above)
Comment (t-h2o) [reply test/src/main.cpp]: done: 9cbf24f1667f6076e31b48d52be3bfed4243a3ed
with this command
```bash
sed -i '' 's/launcher/launch/' $(find . -name "*.cpp" -or -name "*.hpp")
```

**Review by 0xCAF3D0OD — APPROVED** [review]
ok

**Review by RobinBurri — APPROVED** [review]
OK

**Review by 0xCAF3D0OD — APPROVED** [review]
RAD

**Pull request merged**